<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="client">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1031</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 255, 255);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="Enregistrement">
     <attribute name="title">
      <string>Enregistrement</string>
     </attribute>
     <widget class="QPushButton" name="Ajouter">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>330</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cins">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>290</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cin</string>
      </property>
     </widget>
     <widget class="QLabel" name="prenoms">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="noms">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="idproduits">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>200</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id_produit</string>
      </property>
     </widget>
     <widget class="QLabel" name="idclients">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id_client</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Email_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>360</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>nom.prénom@gmail.com</string>
      </property>
     </widget>
     <widget class="QLabel" name="emails">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>370</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="numtels">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>360</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Num_Tel</string>
      </property>
     </widget>
     <widget class="QLabel" name="adresses">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>120</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Prenom_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Vore prénom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Nom_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Votre nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idClient_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>8 chiffres </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="num_3">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>350</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>8 chiffres </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin_3">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>280</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>8 chiffres </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idProduit_3">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>190</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>2A30</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="adresse_3">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>110</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>pays, ville, code postale</string>
      </property>
     </widget>
     <widget class="QPushButton" name="camera">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>470</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera</string>
      </property>
     </widget>
     <widget class="QPushButton" name="play">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>460</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>460</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Consultation">
     <attribute name="title">
      <string>Consultation</string>
     </attribute>
     <widget class="QTableView" name="viewClient">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>961</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="afficher3">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Modification">
     <attribute name="title">
      <string>Modification</string>
     </attribute>
     <widget class="QPushButton" name="Modifier">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Nom">
      <property name="geometry">
       <rect>
        <x>452</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="nomm">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="emaill">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="prenomm">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Prenom">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>30</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="idClientt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> id_client a modifiée</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Email">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="idproduitt">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id_produit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>130</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="cinn">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>130</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Cin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="num">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>180</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="numtell">
      <property name="geometry">
       <rect>
        <x>505</x>
        <y>190</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Num_Tel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idClient">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit (219,199,202)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idProduit">
      <property name="geometry">
       <rect>
        <x>452</x>
        <y>130</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="adressee">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>80</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Adresse">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="viewclient2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>971</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Refrachir">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Réfrachir</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Recherche">
     <attribute name="title">
      <string>Recherche</string>
     </attribute>
     <widget class="QTableView" name="ViewClientRecherche">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>951</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="id_ClientR_1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>id_Client a chercheé</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Recherche_2">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_clientR_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Suppression">
     <attribute name="title">
      <string>Suppression</string>
     </attribute>
     <widget class="QPushButton" name="Supprimer">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_ClientD">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>id_Client a supprimée</string>
      </property>
     </widget>
     <widget class="QTableView" name="viewclient3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>971</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Refrachir_2">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>160</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Réfrachir</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_clientD">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Trier</string>
     </attribute>
     <widget class="QTableView" name="triClient">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>991</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="trier">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>30</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ecxporter_en_pdf">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>70</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter en Pdf</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Email</string>
     </attribute>
     <widget class="QLineEdit" name="server">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>smtp.gmail.com</string>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="port">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>60</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>465</string>
      </property>
      <property name="placeholderText">
       <string>subject</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>971</width>
        <height>241</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>type your message here ...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>80</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="label_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>smtp-server </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="label_14">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>server post </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="paswd">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>140</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="uname">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="label_15">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>user name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="label_16">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>961</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>adresse mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>961</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>object </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Statistique</string>
     </attribute>
     <widget class="QPushButton" name="stat">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>490</y>
        <width>741</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>executer statistique</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>961</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
